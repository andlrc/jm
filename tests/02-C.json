{
	"@extends": ["02-B.json"],
	"myArray": [
		{
			"@prepend": true,
			"c": 100
		},
		{
			"c": 200
		}
	]
}
