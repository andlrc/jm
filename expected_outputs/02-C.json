{
	"myArray2": [
		{
			"b": 10
		}
	],
	"myObject2": {
		"b": 10
	},
	"myArray": [
		{
			"a": 1,
			"b": 10,
			"c": 200
		},
		{
			"a": 2,
			"b": 20
		},
		{
			"c": 100
		}
	],
	"myObject": {
		"prop1": 10,
		"prop2": {
			"b": 20
		},
		"prop3": {
			"a": 3,
			"b": 30
		}
	}
}
